highfidelity/hifi
# Correct desktop icon name

QA Test:
During installation, make sure to create desktop icons. Desktop icon name should be "High Fidelity Interface - PR..." instead of "High Fidelity High Fidelity Interface - PR.."
